submarine-workbench/workbench-server/src/main/resources/org/apache/submarine/database/mappers/TeamMapper.xml

```diff
@@ -82,9 +82,6 @@
       <if test="createTime != null">
         create_time,
       </if>
-      <if test="updateBy != null">
```

Comment:
`updateBy` always record last update user, need update by every times.
All table `createBy` and `createTime` record first user and time.

Reply:
I'm not sure if insert wants to record the value of updateby, or Assign the value of createBy to updateBy.

Reply:
when first insert a record, then the workbench web always set updateby=createby from front-end.
All Table to same.

Reply:
Done.